src/pages/Login/Login.js

```diff
@@ -1,5 +1,91 @@
-import React from 'react';
+/* eslint-disable */
 
-export default function Login() {
-  return <div>hi</div>;
+import React, { useRef, useState } from 'react';
+import { useNavigate } from 'react-router';
+import '../Login/Login.scss';
+import Input from './Input/Input';
+
+function Login() {
+  const idInputRef = useRef();
+  const pwInputRef = useRef();
+  const loginData = {};
+  const [isAlertPopId, setIsAlertPopId] = useState(false);
+  const [isAlertPopPw, setIsAlertPopPw] = useState(false);
+
+  const validateIdBtn = () => {
+    loginData['id'] = idInputRef.current.value;
+    const idValue = loginData['id'];
+    const isIdValid =
+      idValue.includes('@') && idValue.length > 2 ? true : false;
+
+    return isIdValid;
+  };
+  const validatePwBtn = () => {
+    loginData['pw'] = pwInputRef.current.value;
+    const pwValue = loginData['pw'];
+    const isPwValid = pwValue.length > 4 ? true : false;
+
+    return isPwValid;
+  };
+
+  const alertEachValid = () => {
+    if (!validateIdBtn() && validatePwBtn()) {
+      setIsAlertPopId(true);
+      setIsAlertPopPw(false);
+    } else if (validateIdBtn() && !validatePwBtn()) {
+      setIsAlertPopId(false);
+      setIsAlertPopPw(true);
+    } else if (!validateIdBtn() && !validatePwBtn()) {
+      setIsAlertPopId(true);
+      setIsAlertPopPw(true);
+    }
```

Comment:
```suggestion
   setIsAlertPopId(!validateId());
   setPwAlertPopId(!validatePw());
```
함수 자체의 return 값이 validation의 결과값인 true or false이기에 조건식을 계속 추가할 필요 없이 위와같이 표현 가능해보입니다!

Reply:
오호! 리팩토링 리뷰 감사합니다 !